DigitalLearningSolutions.Data.Tests/Services/CourseServiceTests.cs

```diff
@@ -179,14 +188,30 @@ public void
         }
 
         [Test]
-        public void VerifyAdminUserCanAccessCourse_should_return_false_with_incorrect_centre()
+        public void VerifyAdminUserCanManageCourse_should_return_false_with_incorrect_centre()
+        {
+            // Given
+            A.CallTo(() => courseDataService.GetCourseValidationDetails(A<int>._))
+                .Returns((2, 2, false));

```

Comment:
Could we also make it so that the returned course at the incorrect centre is an AllCentre course just to make sure that new addition doesn't break it.